ip/ip.go

```diff
@@ -0,0 +1,20 @@
+package ip
+
+import (
+	"net/http"
+	"strings"
+)
+
+const (
+	headerXForwardedFor = "X-Forwarded-For"
+)
+
+func FromReq(req *http.Request) string {
```

Comment:
We could move it to [httputil](https://github.com/rudderlabs/rudder-go-kit/tree/main/httputil) package, naming it something like `GetRequestIP(req *http.Request) string`

Reply:
Thanks for pointing out. Completely missed out `httputil`.